### ISSUE TITLE
关于训练的疑问

### ISSUE THREAD

**Issue by wjinfeng** [WENGSYX/CirBERTa#5]
```
for step, (input_ids, attention_mask,y) in enumerate(tk):
    with autocast():
        output = model(input_ids=input_ids, input_mask=attention_mask,  labels=y)
        logits = output['logits']
        logits[:,:64243]=0
        logits[:,64244:68158]=0
        logits[:,68159:110093]=0
        logits[:,110094:115270]=0
        logits[:,115271:125883]=0
        logits[:,125884:]=0
        loss = criterion(logits,output['labels'])
        scaler.scale(loss).backward()
```
感谢您分享的baseline代码，关于上述操作我有一点不理解，对logits的部分位置置0的意义是什么，而且上述操作为什么不用`logits[:, 64243:] = 0`代替呢，
希望得到回复，感谢

**Comment by wjinfeng** [comment]
查看了一下，125883、115270、68158、110093、64243分别对应标签坏、差、平、行、好在词表中的id，这里是为了将mlm变成五分类吗？

**Comment by WENGSYX** [comment]
> 查看了一下，125883、115270、68158、110093、64243分别对应坏、差、平、行、好在词表中的id，这里是不是为了将mlm改成五类？

是的，目的相当于不计算其他Token的loss

**Comment by wjinfeng** [comment]
好的，感谢
